qiita_pet/templates/list_studies.html

```diff
@@ -0,0 +1,266 @@
+{% extends sitebase.html %}
+{% block head %}
+<link rel="stylesheet" href="/static/vendor/css/jquery.dataTables.css" type="text/css">
+
+<script src="/static/vendor/js/jquery.dataTables.min.js"></script>
+<script src="/static/vendor/js/jquery.dataTables.plugin.natural.js"></script>
+
+<script type="text/javascript">
+var current_study;
+var ajaxURL = "/study/search/?&user={{current_user.id}}&sEcho=" + Math.floor(Math.random()*1001);
```

Comment:
Ok, attack was wrong word. More just to add a bit of randomness rather than selecting a single number, that way we know the function is actually working and not just spitting back garbage.